ui/app/styles/core/tag.scss

```diff
@@ -52,4 +52,14 @@
   &.is-outlined {
     box-shadow: 0 0 0 1px $white;
   }
+
+  &.is-structure {
+    color: rgb(98, 104, 115);
+    text-transform: none;
+    font-size: 12px;
+
+    width: 48px;
+    height: 16px;
```

Comment:
I'm surprised to see fixed dimensions in a tag. What is this accounting for?

Reply:
That’s how it was specified in Structure. Maybe it’s something I should have ignored. But it’s obsolete now regardless 😆